tests/Filesystem/LocalTest.php

```diff
@@ -23,7 +23,7 @@ public function testReplicate(): void
         $local = new Local('/tmp');
 
         // check if OS is Mac OS X where /tmp is a symlink to /private/tmp
-        $result = 'Darwin' === php_uname('s') ? '/private/tmp' : '/tmp';
+        $result = 'Darwin' === PHP_OS ? '/private/tmp' : '/tmp';
```

Comment:
I wonder if this could be simplified by using `sys_get_tmp_dir()`

Reply:
mmm using `sys_get_temp_dir` with Mac OS X I get something like `/var/folders/48/fkmnn6gs2998gq8wc5f83d780000gn/T`

Reply:
Ok, doesn't look like it…